client/src/Components/NavigationBar.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+
+import { makeStyles } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+
+import Button from '@material-ui/core/Button';
+import Typography from "@material-ui/core/Typography";
+import DashboardRoundedIcon from '@material-ui/icons/DashboardRounded';
+import CalendarTodayOutlinedIcon from '@material-ui/icons/CalendarTodayOutlined';
+import AddIcon from '@material-ui/icons/Add';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import IconButton from '@material-ui/core/IconButton';
+
+const useStyles = makeStyles(theme => ({
+    iconButtonLabel: {
+     display: 'flex',
+    },
+    appBar: {
+      background: "#FFFFFF",
+      boxShadow: "none",
+      display: 'flex',
+      flexDirection: "row",
+      alignItems: 'center',
+      justifyContent: 'space-between',
+    },
+    toolBar: {
+      justifyContent: 'center',
+      alignItems: 'center',
```

Comment:
the position is not placed rightly, you can maybe try using the position, top, right, left, bottom css attributes if that is more accurate

Reply:
This was fixed when I placed all the elements in a single toolbar